pkg/virt-controller/services/multus_annotations.go

```diff
@@ -75,6 +88,22 @@ func GenerateMultusCNIAnnotationFromNameScheme(namespace string, interfaces []v1
 	return "", nil
 }
 
+func newBindingPluginMultusAnnotationData(kvConfig *v1.KubeVirtConfiguration, pluginName, namespace, networkName string) (*networkv1.NetworkSelectionElement, error) {
+	plugin := netbinding.ReadNetBindingPluginConfiguration(kvConfig, pluginName)
+	if plugin == nil {
+		return nil, fmt.Errorf("couldnt find network binding plugin '%s' in Kubevirt configuration", pluginName)
```

Comment:
nit: `couldn't` or `could not` or my preferred one: `unable to find the network ...`

Reply:
Done